app/components/ExpirationInput/ExpirationInput.css.js

```diff
@@ -0,0 +1,40 @@
+import { StyleSheet } from 'react-native';
+import { Colors } from '../../theme/theme';
+
+export const ExpirationInputCSS = StyleSheet.create({
+  expirationInput: {
+    flexDirection: 'column',
+  },
+  expirationToggleRow: {
+    flexDirection: 'row',
+    paddingRight: 16,
+    justifyContent: 'space-between',
+    height: 48
+  },
```

Comment:
I was trying to follow material design specs. It was my intention to have the row remain the same height with the same padding across devices. That being said, I haven't looked at it on a tablet, but I think it will look ok. If we need to we can update this later.